scripts/generate-tags.sh

```diff
@@ -18,6 +18,10 @@ if [ -f /var/log/hardening ]; then
     TAGS=$(echo $TAGS | jq -r --arg hardening "$HARDENING" '. + {"hardening": $hardening}')
 fi
 
+if [[ "$STIG_ENABLED" == true ]]; then
```

Comment:
This will be handled by the previous hardening file

Reply:
obsolete